### ISSUE TITLE
SQL select statement, with big numbers

### ISSUE THREAD

**Issue by pgulutzan** [tarantool/tarantool#2363]
SELECT a big number, and it will not be displayed correctly.
However, it is clear that it is stored correctly, because a NoSQL
request displays the right value. Example:
```
tarantool> box.sql.execute("create table t7 (s1 int primary key, s2 char)")
---
...

tarantool> box.sql.execute("insert into t7 values (123456789012,'Hello')")
---
...

tarantool> box.sql.execute("select * from t7")
---
- - [-1097262572, 'Hello']
...

tarantool> box.space.t7:select()
---
- - [123456789012, 'Hello']
...
```

**Comment by kyukhin** [comment]
This was duplicate of #2469, looks like resolved now, right?

**Comment by Khatskevich** [comment]
fixed in 2420040b1787b55dddebd52f58567f358d007b9b
